src/test/java/seedu/address/logic/commands/AddOrderCommandTest.java

```diff
@@ -31,7 +33,8 @@ public void execute_personAcceptedByModel_addSuccessful() throws Exception {
                 .ModelStubAcceptingOrderAdded();
         Order validOrder = new OrderBuilder().build();
 
-        CommandResult commandResult = new AddOrderCommand(validOrder).execute(modelStub);
+        CommandResult commandResult = new AddOrderCommand(new Details(OrderBuilder.DEFAULT_DETAILS),
+                new Phone(OrderBuilder.DEFAULT_PHONE)).execute(modelStub);
```

Comment:
addOrderCommand's execute method will throw an AssertionError because it takes in modelStub as the input, and modelStub's hasOrder method will throw that exception